src/amo/reducers/addonsByAuthors.js

```diff
@@ -153,7 +154,9 @@ export const getAddonsForUsernames = (
       return state.byAddonId[id];
     })
     .filter((addon) => {
-      return addonType ? addon.type === addonType : true;
+      // return addonType ? addon.type === addonType : true;
```

Comment:
Can we remove this line instead of commenting it?

Reply:
yes that was not intended :/